src/controllers/apiContentController.js

```diff
@@ -1,31 +1,40 @@
+/* eslint-disable no-undef */
 const { renderTemplate, renderTemplateFromAPI, renderGivenTemplate, loadLayoutFromAPI, loadMarkdown } = require('../utils/util');
-const config = require('../config/config');
-const markdown = require('marked');
+const config = require(process.cwd() + '/config.json');
 const fs = require('fs');
 const path = require('path');
 const exphbs = require('express-handlebars');
+const constants = require('../utils/constants');
+const adminDao = require('../dao/admin');
+const apiDao = require('../dao/apiMetadata');
+const apiMetadataService = require('../services/apiMetadataService');
 
-
-filePrefix = '../../../../src/'
+const filePrefix = config.pathToContent;
 const generateArray = (length) => Array.from({ length });
-let baseURL = "http://localhost:" + config.port;
 
 const loadAPIs = async (req, res) => {
 
     const orgName = req.params.orgName;
-    let metaData = await loadAPIMetaDataList(orgName);
     let html;
-    if (config.mode == 'design')
-        baseURL = orgName;
-
-    let templateContent = {
-        apiMetadata: metaData,
-        baseUrl: baseURL
-    }
-    if (config.mode == 'single' || config.mode == 'design') {
+    if (config.mode === constants.DEV_MODE) {
+        let metaData = await loadAPIMetaDataList()
+        let templateContent = {
+            apiMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
         html = renderTemplate(filePrefix + 'pages/apis/page.hbs', filePrefix + 'layout/main.hbs', templateContent);
     } else {
-        html = await renderTemplateFromAPI(templateContent, orgName, "apis");
+        let organization = await adminDao.getOrganization(orgName);
```

Comment:
use `const` instead of `let` wherever possible.

Reply:
changed